modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java

```diff
@@ -594,4 +605,89 @@ public void testInsertMultipleDefaultsWithImplicitPk() {
                 .returns(5, 2)
                 .check();
     }
+
+    @ParameterizedTest
```

Comment:
I think the test should be in another place, for example together with antoher *View tests